mailpoet/lib/Cron/Workers/Scheduler.php

```diff
@@ -332,49 +340,56 @@ private function processReEngagementEmail($queue) {
     return true;
   }
 
-  public function verifyMailpoetSubscriber($subscriberId, $newsletter, $queue) {
-    $subscriber = Subscriber::findOne($subscriberId);
+  public function verifyMailpoetSubscriber($subscriberId, NewsletterEntity $newsletter, $queue) {
+    if (is_int($subscriberId)) {
+      $subscriber = $this->subscribersRepository->findOneById($subscriberId);
+    } else {
+      $subscriber = null;
+    }
```

Comment:
From a quick browse through the code, I think we could do the following:
```suggestion
  public function verifyMailpoetSubscriber(int $subscriberId, NewsletterEntity $newsletter, $queue) {

```